Hidpi support

PegasisForever opened the pull request:
- Added a new setting to set scaling of the application.
- Automatically scale font size.

peerdavid commented:
Thanks for this great improvement! I will test your solution to hidpi soon and merge the PR afterward.

peerdavid commented:
I tested this PR but I have some problems: I changed the resolution to 100% and then nothing is shown in the File Explorer and also not in the Settings. So I can not change it back. Also, the text looks relatively small in the default of 75% on my machine when I start RemaPy the first time. I use a display with a resolution of 1920 x 1080.

PegasisForever commented:
> I changed the resolution to 100% and then nothing is shown in the File Explorer and also not in the Settings.

Strange, it works fine on my machine, can you check what value of `scaling` got saved to the `~/.remapy/config` file?

>  Also, the text looks relatively small in the default of 75% on my machine

The default scaling comes from `window.tk.call('tk', 'scaling')`, which to my understanding, doesn't respect dpi settings of the system and returns `screen physical dpi / 72`. I changed it so it defaults to 100% now.

sabrekke commented:
I'm using RemaPy on a Hidpi display, and just changed the parameters in rema.py manually to increase the row and icon size. This worked well for a few weeks, then suddenly today the file explorer failed to render as described by peerdavid. The settings worked as normal, as I had not made any changes there. 

Not sure if this is of any help at all, but at least I encountered the same issues independently of this pull request.
